.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: u11d-bartlomiej-galezowski, u11d-michal-miler
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. 
+
+**Expected behavior**
```

Comment:
1. What do you think about additional `Current behavior` section?
2. I think we should check MD files by Prettier (empty line between lists or paragraph sections for instance)

```suggestion
**Current behavior**

A clear and concise description of current behavior.

**Expected behavior**

A clear and concise description of what you expected to happen.
```

Reply:
1. Good idea.
2. Yes, we should do this. I will create an issue when templates will merged.